openmc/capi/filter.py

```diff
@@ -253,4 +255,12 @@ def __len__(self):
     def __repr__(self):
         return repr(dict(self))
 
+
 filters = _FilterMapping()
+
+
+def get_free_filter_id():
```

Comment:
What do you think about making this a `@classmethod` of `capi.Filter` called `next_id`? That way, it would kind of match the machinery used for auto-assigning IDs via `IDManagerMixin`.

Reply:
Actually, come to think it, we already handle this on the Python side (see [here](https://github.com/mit-crpg/openmc/blob/develop/openmc/capi/filter.py#L66-L76)) so I don't think we really need a Python binding to the Fortran subroutine.

Reply:
Good point.  I've removed the Python binding